gen/calc.py

```diff
@@ -970,6 +970,7 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         lambda fault_domain_enabled: validate_true_false(fault_domain_enabled),
         lambda mesos_master_work_dir: validate_absolute_path(mesos_master_work_dir),
         lambda mesos_agent_work_dir: validate_absolute_path(mesos_agent_work_dir),
+        lambda dcos_diagnostics_bundles_dir: validate_absolute_path(dcos_diagnostics_bundles_dir),
```

Comment:
This is a user-facing config param, so I have to be picky about it. Let's drop the `dcos_` since that's redundant: `diagnostics_bundles_dir`.